l10n_it_partially_deductible_vat/__openerp__.py

```diff
@@ -24,27 +24,17 @@
 ##############################################################################
 
 {
-    "name" : "Italy - Partially Deductible VAT",
-    "version" : "0.1",
-    "depends" : ['account'],
-    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
-    "description": """
-    Calcoli aggiuntivi per la gestione dell'IVA parzialmente detraibile. Senza questo modulo non è possibile calcolare correttamente imponibili, imposte e conti imposte relativi all'IVA parzialmente detraibile.
-    La configurazione dell'IVA parzialmente detraibile deve essere quella utilizzata dal modulo 'l10n_it', anche se non viene usato tale modulo.
-    """,
+    "name": "Italy - Partially Deductible VAT",
+    "version": "0.1",
+    "depends": ['account'],
+    "author": "OpenERP Italian Community,Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "category" : "Localisation/Italy",
-    'website': 'http://www.openerp-italia.org/',
-    'init_xml': [
-        ],
-    'update_xml': [
-        ],
-    'demo_xml': [
-        ],
+    "category": "Localisation/Italy",
+    'website': 'http://www.odoo-italia.org/',
     'test': [
         'test/account_tax.xml',
         'test/tax_computation.yml',
-        ],
-    'installable': False,
+    ],
+    'installable': True,
     'active': False,
```

Comment:
This key is deprecated and it's an alias for auto_install whose default value is already False. So please remove it.